app/models/chargeback_rate.rb

```diff
@@ -72,36 +74,67 @@ def self.seed_chargeback_rate_measure
     end
   end
 
+  def self.seed_cb_rate_currency
```

Comment:
Please name this method `seed_chargeback_rate_detail_currency`...  Or use the belongs_to name: `seed_details_currency`

I don't see much value to abbreviating if we're making it harder to grep the code.